Kubernetes/omsagent.yaml

```diff
@@ -571,6 +571,103 @@ spec:
           configMap:
             name: container-azm-ms-agentconfig
             optional: true
+---
+  apiVersion: apps/v1
+  kind: DaemonSet
+  metadata:
+   name: omsagent-win
+   namespace: kube-system
+   labels:
+    component: oms-agent
+    tier: node
+  spec:
+   updateStrategy:
+    type: RollingUpdate
+   selector:
+    matchLabels:
+      component: oms-agent
+      tier: node
+   template:
+    metadata:
+      labels:
+        component: oms-agent
+        tier: node
+      annotations:
+        agentVersion: "1.10.0.1"
+        dockerProviderVersion: "9.0.0-1"
+        schema-versions: "v1"
+    spec:
+     serviceAccountName: omsagent
+     containers:
+       - name: omsagent-win
+         image:  mcr.microsoft.com/azuremonitor/containerinsights/cidev:win-20200507-062708z-7f95982
+         imagePullPolicy: IfNotPresent
+         resources:
+          limits:
+           cpu: 150m
+           memory: 600Mi
+          requests:
+           cpu: 75m
+           memory: 225Mi
+         env:       
+          # - name: AKS_RESOURCE_ID
+          #   value: "VALUE_AKS_RESOURCE_ID_VALUE"
+          # - name: AKS_REGION
+          #   value: "VALUE_AKS_RESOURCE_REGION_VALUE"
+          - name: ACS_RESOURCE_NAME
+          value: "my_acs_cluster_name"
+          - name: CONTROLLER_TYPE
+            value: "DaemonSet"
+          - name: CI_CERT_LOCATION
+            value: "C://oms.crt"
```

Comment:
since these (crt and key) are not customer configurable so these should be moved to Docker File as well.